### PR TITLE
fix: action agent support set file stream timeout by env

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
action agent support set file stream timeout by env

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/ticket?id=310784)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | Fix the bug that action agent support set file stream timeout by env             |
| 🇨🇳 中文    |   action agent支持配置文件流的超时时间           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/actionagent/callback.go [hunk 1]
```diff
@@ -80,18 +85,30 @@ func (agent *Agent) SetCallbackReporter() {
 		agent.AppendError(err)
 		return
 	}
+	var fileStreamTimeoutSec uint64 = 60
```
Comment (sfwn) [inline modules/actionagent/callback.go:88]: parse env at prepare step

Comment (sfwn) [reply modules/actionagent/callback.go:88]: also put into easy use

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/actionagent/callback.go (hunk 1 above)
Comment (chengjoey) [reply modules/actionagent/callback.go:88]: still parse env at `SetCallbackReporter`, and put into agent eay use

**Review by sfwn — APPROVED** [review]

**Pull request merged**